src/test/java/core/basesyntax/LambdaCalculatorTest.java

```diff
@@ -0,0 +1,127 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class LambdaCalculatorTest {
+    private static LambdaCalculator calculator;
+
+    @BeforeAll
+    static void beforeAll() {
+        calculator = new LambdaCalculator();
+    }
+
+    @Test
+    void testForAddition() {
+        double firstActual = calculator.calculate(Integer.MAX_VALUE, Integer.MAX_VALUE, '+');
+        double secondActual = calculator.calculate(20, 15, '+');
+        double thirdActual = calculator.calculate(-100, 50, '+');
```

Comment:
```suggestion
        double secondActual = calculator.calculate(20, 15, '+');
        double thirdActual = calculator.calculate(-100, 50, '+');
```
we have the same value three times in test, we can add it on beforeAll annotation